### ISSUE TITLE
Exemplar pop-up sprawls off screen and cannot be scrolled

### ISSUE THREAD

**Issue by tcolgate** [grafana/grafana#127848]
**Steps to reproduce**:

- On a lower res display with larger fonts (e.g. 1024x768 with a 12pt font), set up a chart with exemplars that have a large number of labels. (preferably enable trace id linking to the trave view)
- Mouse over a given exemplar
- Attempt to interact with the resulting pop-up panel.

**Actual Result**:
- The bottom of the pop-up is usually off screen
- The button to link to the trace view will be off the screen and cannot be interacted with
- it is not possible to scroll the panel, or other move down to see off screen info
- depending on the position of hte panel and the size of the exemplar list, the user my "zoom out" to fit everything
  on, beyond a certain point it is no longer possible to highlight the exemplar to bring the popup up anyway.

**Expected Result**
- it should be possible to scroll and navigate the info in the panel even if it is too big to fit on screen.
- it should be possible to jump to the linked trace view even if there is a lot of into in the panel
- 
**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: Grafana v12.3.1 (3a1c80ca7c)
- Data source type & version: Prometheus
- User OS & Browser: Google Chrome on Linux
- Others:

**Comment by AnuragRaut08** [comment]
Hi! I'd like to work on this issue.

I'll first reproduce the problem locally and investigate the exemplar pop-up implementation to identify the appropriate fix. My aim is to ensure the pop-up remains fully usable when its content exceeds the available viewport height, so users can scroll through all content and access the trace-view link without affecting the existing behavior for normal-sized pop-ups.

I'll follow up with a PR once I have a solution.

**Comment by ashharrison90** [comment]
@grafana/dataviz-squad believe this is your remit - shout if that's incorrect 🙏

**Comment by AnuragRaut08** [comment]
Thanks for the routing! I’ll investigate this on the latest `main` and follow up with a PR if I’m able to resolve it.
